app/models/device_problem.rb

```diff
@@ -0,0 +1,11 @@
+class DeviceProblem < ApplicationRecord
+
+  belongs_to :device
+
+  validates :problem, :content, :device_id, presence: true
+
+  def to_s
+    problem
```

Comment:
I don't know if this name is super clear as you'll be calling something like `device_problem.problem`.  Would something like `device_problem.statement` or `device_problem.headline` make any more sense?